frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelTab.tsx

```diff
@@ -130,23 +130,24 @@ export function FunnelTab(): JSX.Element {
                         <TestAccountFilter filters={filters} onChange={setFilters} />
                     </div>
                 </div>
-                <PropertyFilters
-                    pageKey={`EditFunnel-property`}
-                    propertyFilters={filters.properties || []}
-                    onChange={(anyProperties) => {
-                        setFilters({
-                            properties: anyProperties.filter(isValidPropertyFilter),
-                        })
-                    }}
-                    taxonomicGroupTypes={[
-                        TaxonomicFilterGroupType.EventProperties,
-                        TaxonomicFilterGroupType.PersonProperties,
-                        ...groupsTaxonomicTypes,
-                        TaxonomicFilterGroupType.Cohorts,
-                        TaxonomicFilterGroupType.Elements,
-                    ]}
-                    eventNames={allEventNames}
-                />
+                {filters.properties && (
+                    <PropertyGroupFilters
+                        propertyFilters={convertPropertiesToPropertyGroup(filters.properties)}
```

Comment:
We will be removing the old property filters entirely for the global filters on `trend tab, funnel tab, etc` and implement the new property group filters. Any old properties will be converted to property group formatting here